tests/selenium/conftest.py

```diff
@@ -376,9 +376,7 @@ def test(request, dash_thread_server, tmpdir):
         remote_url=request.config.getoption("remote_url"),
         headless=request.config.getoption("headless"),
         options=request.config.hook.pytest_setup_options(),
-        download_path=tmpdir.mkdir("download").strpath,
-        percy_assets_root=request.config.getoption("percy_assets"),
+        download_path=tmpdir.mkdir("dt-download").strpath,
```

Comment:
Table tests sometimes use `dash_thread_server` and `dash_duo` -- `dash_duo` specifically also creates a temporary folder, both fixtures clash and the tests end up throwing an error because the directory already exists. Pytest uses a py27 compatible object that does not expose `exists_ok` and hence the error can't otherwise be prevented.